spec/system/admin/order_cycles/simple_spec.rb

```diff
@@ -368,6 +368,43 @@
         end
       end
 
+      context "when variants are hidden via inventory settings" do
+        let(:oc) do
+          create(:simple_order_cycle, suppliers: [supplier_managed],
+                                      coordinator: distributor_managed,
+                                      distributors: [distributor_managed],
+                                      name: 'Order Cycle 1' )
+        end
+
+        let(:v1) { create(:variant, product: create(:product, supplier: supplier_managed) ) }
+        let(:inventory_item) {
+          create(:inventory_item, enterprise: distributor_managed, variant: v1, visible: false)
+        }
+        before do
+          # Incoming exchange
+          ex_in = oc.exchanges.where(sender_id: supplier_managed, receiver_id: distributor_managed,
+                                     incoming: true).first
+          ex_in.update(variant_ids: [v1.id])
+
+          # Outgoing exchange
+          ex_out = oc.exchanges.where(sender_id: distributor_managed,
+                                      receiver_id: distributor_managed, incoming: false).first
+          ex_out.update(variant_ids: [v1.id])
+
+          # hide via inventory settings variant v1
+          ## HOW TO?
```

Comment:
I think what you're looking for is:
`oc.update prefers_product_selection_from_coordinator_inventory_only: true`

Reply:
I tried (see my last commit, squashed: 091939a64f18f90c1f879effcb8d15873a5be18c), but I failed to reproduce the case where no variant are present (because hidden via inventory) for a product in "Outgoing Products" section.

Would you like to have a look @filipefurtad0 ?

Reply:
If we disable this option, we can see all the variants:
![image](https://github.com/openfoodfoundation/openfoodnetwork/assets/49817236/2e590847-1de7-4f85-aa78-e785c6fcf087)

With the Coordinators inventory only, we get:
![image](https://github.com/openfoodfoundation/openfoodnetwork/assets/49817236/db745d19-b6d3-4bd1-9714-9e157183ffc9)

So, I think the solution to have no variants preset on the OC is to hide it, when `inventory_item_v3` is being created a few lines above (by setting it to `visible: false`):


```
        let(:inventory_item_v3) {
          create(:inventory_item, enterprise: distributor_managed, variant: v3, visible: false)
        }
```

Doing so:

![image](https://github.com/openfoodfoundation/openfoodnetwork/assets/49817236/f6959bc3-f430-4ed7-a0a2-6ec4178ccbf9)

Does this help?

Reply:
How is your enterprise configured? Do you have this preference enabled?

![image](https://github.com/openfoodfoundation/openfoodnetwork/assets/49817236/97c55c0e-7030-4ad6-8c27-e0c134b9d767)

Reply:
On the `before` block, if we add:

`supplier_managed.update preferred_product_selection_from_inventory_only: true`

and change the OC setting to:
`oc.update prefers_product_selection_from_coordinator_inventory_only: false`

Then we get:

![image](https://github.com/openfoodfoundation/openfoodnetwork/assets/49817236/6f70533b-0b9a-4346-93a2-77457bff7ccc)

Which makes the spec pass :green_circle:

Reply:
ohhhhh Thanks @filipefurtad0 !